index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
+
+## Start with the README
+Your application’s README is still its gateway, the first thing new developers will likely read when they access its source code. If your app’s README consists of the default given to every new Rails app, it’s time to change it.
+
+Your README should include:
+
+* A brief description of what the application does.
+* How to install dependencies, especially those that are more complex than a simple `bundle install`. For example, if I need to to install Redis or ImageMagick to work on the app, or configure environment variables to talk to external services, let me know up front.
+* How to run the application’s test suite, to verify that setup was successful. Again, note special external dependencies, like PhantomJS.
+* How to generate the docs, to get to a more detailed description of the application’s code.
```

Comment:
At one point @pcullen-nerdery, Collin, and I all sat around talking about standardizing "project level" documentation. We actually got a whole bunch of folks in a room from around the company and tried to agree on standardizing something.... I should have known better that wasn't going to end well... and it didn't, just went to the wayside.

That being said, I think there's a lot to be said for doing it team level and trying to propagate it to the rest of the company later down the line. I very much would be interested in setting up a standardization around this. I think some other items that would be well shimmed in between your first and second bullet point (somewhat covered in your last bullet):

- Team Members (Developers, PMs, SEs, ect.) with their email and their role on the project
- Communication / Issue Tracking links
- Link to Mainframe systems page
- External Documentation links (link tot he drive folder, any proprietary API's, ect.)
- Branching Strategy
- Any specific "coding" standards specific to the project